service/courseService/courseService.go

```diff
@@ -37,3 +37,29 @@ func (cS CourseService) GetCourse(id string, cef eduboard.CourseEntryManyFinder)
 	course.Entries = e
 	return nil, course
 }
+
+func (cS CourseService) GetCoursesByMember(id string) (error, []eduboard.Course) {
+	return cS.CR.FindByMember(id)
+}
+
+func (cS CourseService) GetMembers(id string, uF eduboard.UserFinder) (error, []eduboard.User) {
+	err, course := cS.CR.FindOneByID(id)
+	if err != nil {
+		return err, []eduboard.User{}
```

Comment:
Please wrap errors with some additional context here. You can see `courseService` or `courseEntryService` for some examples.